src/main/java/algorithms/curated170/medium/numberofconnectedcomponents/NumberOfConnectedComponentsUnionFind.java

```diff
@@ -0,0 +1,51 @@
+package algorithms.curated170.medium.numberofconnectedcomponents;
+
+public class NumberOfConnectedComponentsUnionFind {
+
+    int roots[];
+
+    public int countComponents(int[][] edges, int n) {
```

Comment:
Can we alter the params order:
public int countComponents(int n, int[][] edges)

Reply:
All fixed.

Reply:
All fixed.